src/code.cloudfoundry.org/vxlan-policy-agent/enforcer/enforcer.go

```diff
@@ -211,23 +210,12 @@ func (e *Enforcer) cleanupOldRules(logger lager.Logger, table, parentChain, mana
 					return err
 				}
 			}
-		} else {
-			if cleanupParentChain {
-				matches := reOtherRules.FindStringSubmatch(r)
-
-				if len(matches) > 1 {
-					logger.Debug("cleaning-up-parent-ruleset", lager.Data{"name": matches[0]})
-					rule, err := rules.NewIPTablesRuleFromIPTablesLine(matches[1])
-					if err != nil {
-						return fmt.Errorf("parsing parent chain rule: %s", err)
-					}
-					logger.Debug("removing-rule", lager.Data{"rule": rule})
-					err = e.iptables.Delete(table, parentChain, rule)
-					if err != nil {
-						return fmt.Errorf("clean up parent chain: %s", err)
-					}
-				}
-			}
+		}
+	}
+	if cleanupParentChain {
+		err := e.iptables.DeleteAfterRuleNum(table, parentChain, 2)
```

Comment:
Can you add a comment here about why you only want to delete after 2.

Reply:
👍